code/modules/clothing/masks/stone.dm

```diff
@@ -82,3 +74,20 @@
 
 /obj/item/clothing/mask/stone/infinite //this mask can be used any number of times
 	infinite = 1
+
+/proc/makeLateVampire(var/mob/living/carbon/human/H, var/blood_to_give)
+	var/datum/role/vampire/vamp =  new(H.mind, override = TRUE)
+	var/datum/faction/vampire/Fac_vamp = new(vamp)
+	if (!vamp || !Fac_vamp)
+		return FALSE
+	ticker.mode.factions += Fac_vamp
+	ticker.mode.orphaned_roles -= vamp
+	vamp.OnPostSetup()
+	log_admin("[H] has become a vampire using a stone mask.")
```

Comment:
Doesn't strictly become a vampire using the stone mask using this proc. Move the log_admin to where it's being called in this case.